gobase/intermediate.go

```diff
@@ -72,7 +72,7 @@ func SaveArr(path *string, dataArr []string) bool {
 		panic(err)
 	}
 	defer file.Close()
-	var dataString = "["
+	var dataString = "string["
```

Comment:
please remove this. Since, when it is string, the Unicode need not be there, and the retrival is made accordingly.